.lgtm.yml

```diff
@@ -0,0 +1,12 @@
+# Configure LGTM for this package
+
+extraction:
+  python:  # Configure Python
+    python_setup:  # Configure the setup
+      version: 3  # Specify Version 3
+path_classifiers:
+  library:
+    - versioneer.py  # Set Versioneer.py to an external "library" (3rd party code)
```

Comment:
Why versioneer? I thought it was dropped. Can it be removed?

I don't know what lgtm is good for and why versioneer is mentioned here.

Reply:
I don't know either. I have removed it. I suspect that they forget to remove it.